### PR TITLE
add commands for lecture 10

### PR DESCRIPTION
**Pull request by MarkWarneke**
- add my lecture notes

### CODE REVIEW

**Review by nyanhp — APPROVED** [review]
looks good!

File: lecture-10/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Lecture 10
+
+git clone ...
```
Comment (MarkWarneke) [inline lecture-10/README.md]: If think we should add the url here @nyanhp

Comment (nyanhp) [reply lecture-10/README.md]: 🥳 Sure!

Comment (MarkWarneke) [reply lecture-10/README.md]: got it!

**Pull request merged**